### ISSUE TITLE
Use system font in tables

### ISSUE THREAD

**Issue by martinbean** [sequelpro/sequelpro#2457]
Although the MySQL content font is configurable, it would be nice if it were the system font (San Francisco) by default rather than Lucida Grande, which looks a bit dated in later OS X versions.

**Comment by stuconnolly** [comment]
Need to see what the oldest OS version we're going to be supporting in the next release as San Francisco has only been available since 10.11 (and it's likely we'll be supporting versions before this).

The only option seems to be removing the hard coded Lucida Grande from the preference defaults and set it when the app first launches to the default system font.

**Comment by martinbean** [comment]
@stuconnolly I’m not really _au fait_ with macOS development, but can the table font just use the system default (so San Francisco in 10.11+, Lucida Grande for below), and only set it if the user’s specified a custom font? I think this is how iOS apps works across iOS versions.

**Comment by stuconnolly** [comment]
Yep, was more concerned about the hard coding of the font in our preference defaults, but think we can get around that.
